packages/veto-frontend/src/api/index.ts

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios'
+
+export const getNewAddress = async (): Promise<string> => {
+  const { data }: { data: { address: string } } = await axios(`http://${window.location.host}/api/getnewaddress`)
```

Comment:
I think you can do ` const { data } = await axios.get<Promise<{data: { address: string }}>>('http://${window.location.host}/api/getnewaddress')` and you wont need the extra type definition for axios